spec/disk_spec.rb

```diff
@@ -37,7 +37,7 @@
 
       disk = LinuxAdmin::Disk.new :path => '/dev/hda'
       allow(disk).to receive(:run!).and_return(double(:output => fdisk))
-      expect(disk.size).to eq(500.1.gigabytes)
+      expect(disk.size).to eq(500107862016)
```

Comment:
OHHHHH I totally misunderstood...you changed it to just return bytes, so we just use the value directly.